apps/modernization-ui/src/components/FormInputs/DatePickerInput.tsx

```diff
@@ -22,6 +24,7 @@ type DatePickerProps = {
     required?: boolean;
     disabled?: boolean;
     disableFutureDates?: boolean;
+    control?: Control<PersonFilter, any>;
```

Comment:
issue (blocker):  This is a general component and should not include code to address specific features.

Reply:
This is not code specific, its generic code for the component. And wherever this component is if there is a situation to clear the Date picker input or its error message its undoable so it needs to be the feature of this component but not the feature for Advance search.

Reply:
I tried but it is notorious, setValue doesn't update the defaultValue but does trigger the control's watch. 
If you click the "Clear All" button next time only then the defaultValue is in the context.

Reply:
The `defaultValue` is already available to the `DatePickerInput`.  The state of `error` is defined by `!(emptyDefaultValue || validDefaultValue)` including `defaultValue === 'none'` should set the error state without needing the control in the component.

My aversion to adding the `control` as a prop to this component is that it's adding Patient Search code into a component used throughout the system.

Reply:
Understood about the control being a prop as a problem. Made the change to insert the defaultValue check in the useState.